symplyphysics/laws/dynamics/buoyant_force_from_density_and_volume.py

```diff
@@ -8,42 +8,33 @@
 """
 
 from sympy import (Eq, solve)
-from symplyphysics import (clone_as_symbol, symbols, units, Quantity, Symbol, validate_input,
+from symplyphysics import (clone_as_symbol, symbols, quantities, Quantity, validate_input,
     validate_output)
+from symplyphysics.core.functions import mul
 
 buoyant_force = clone_as_symbol(symbols.force,
-    display_symbol="Fa",
+    display_symbol="F_A",
     display_latex="F_\\text{A}")
 """
 The buoyant (Archimedes) :symbols:`force`.
 """
 
-fluid_density = Symbol("fluid_density", units.mass / units.volume)
-r"""
-The density of the fluid.
-
-Symbol:
-    rho
-
-Latex:
-    :math:`\rho`
+fluid_density = symbols.density
 """
-
-displaced_volume = Symbol("displaced_volume", units.volume)
+The :symbols:`density` of the fluid.
 """
-The volume of the displaced fluid. Equivalently, the volume of the part of the body immersed in the fluid.
 
-Symbols:
-    V
+displaced_volume = symbols.volume
+"""
+The :symbols:`volume` of the displaced fluid. Equivalently, the volume of the part of the body
+immersed in the fluid.
 """
 
-law = Eq(buoyant_force, -1 * fluid_density * units.acceleration_due_to_gravity * displaced_volume)
-r"""
-Fa = -1 * rho * g * V
+law = Eq(buoyant_force, mul(-1, fluid_density, quantities.acceleration_due_to_gravity, displaced_volume))
```

Comment:
Because I wanted the order of the symbols to remain the same, somewhat following the principle of least astonishment regarding the user. Although the LaTeX printer still does some re-ordering afterwards so it only worked for the code printer.

Reply:
I will look into this issue. But `mul` is really ugly. Avoid this solution.

Reply:
Okay